api/api/create_pit.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ */
+
+'use strict';
+
+/* eslint camelcase: 0 */
```

Comment:
What is the meaning of this two lines of eslint rules? could you provide some explains for adding these two lines? for example, is this `eslint camelcase: 0` same as `esline-disable camelcase` which disable camelcase check for a file?

Reply:
This is just to keep it consistent with the other files, so that once we fix the auto-generation logic, we won't see any changes in these files.

Reply:
Well I think I don't understand these two lines. Do you understand why we use these two lines?

Reply:
AFAIK `eslint camelcase: 0` and `eslint no-unused-vars: 0` are for disabling camelcase check and unused variables check respectively.